src/test/java/com/wikia/webdriver/testcases/mercurytests/recentwikiactivity/RecentWikiActivityTest.java

```diff
@@ -0,0 +1,48 @@
+package com.wikia.webdriver.testcases.mercurytests.recentwikiactivity;
+
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.mercury.pages.RecentWikiActivityPage;
+
+import org.testng.annotations.Test;
+
+@Execute(onWikia = "aga")
+@InBrowser(
+    browser = Browser.CHROME,
+    emulator = Emulator.GOOGLE_NEXUS_5
+)
+public class RecentWikiActivityTest extends NewTestTemplate {
+
+  @Test()
+  public void recentWikiActivityAsAnon_redirectionToDiffPageAndGoingBack() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)